apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/SoeknadInformasjon.tsx

```diff
@@ -0,0 +1,82 @@
+import { IDetaljertBehandling } from '~shared/types/IDetaljertBehandling'
+import { usePersonopplysninger } from '~components/person/usePersonopplysninger'
+import { BodyShort, Detail, Table, VStack } from '@navikt/ds-react'
+import { formaterDatoMedFallback } from '~utils/formatering/dato'
+
+export const SoeknadInformasjon = ({ behandling }: { behandling: IDetaljertBehandling }) => {
+  const personopplysninger = usePersonopplysninger()
+
+  const visOmInnsenderErGjenlevendeEllerVerge = (): string => {
+    // Innsender er gjenlevende
+    if (
+      personopplysninger?.innsender?.opplysning.foedselsnummer === personopplysninger?.soeker?.opplysning.foedselsnummer
+    ) {
+      return '(gjenlevende)'
+    }
+
+    // Innsender er verge
+    if (
+      personopplysninger?.innsender?.opplysning.foedselsnummer ===
+      personopplysninger?.soeker?.opplysning.vergemaalEllerFremtidsfullmakt?.[0].vergeEllerFullmektig
+        .motpartsPersonident
+    ) {
+      return '(verge)'
+    }
+
+    return ''
+  }
+
+  return (
+    <Table size="small">
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell scope="col">Søknad mottat</Table.HeaderCell>
+          <Table.HeaderCell scope="col">Innsender</Table.HeaderCell>
+          <Table.HeaderCell scope="col">Foreldreansvar</Table.HeaderCell>
+          <Table.HeaderCell scope="col">Vergemål</Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        <Table.Row>
+          <Table.DataCell>{formaterDatoMedFallback(behandling.soeknadMottattDato)}</Table.DataCell>
+          <Table.DataCell>
+            {!!personopplysninger?.innsender ? (
+              <VStack>
+                <BodyShort>{`${personopplysninger.innsender.opplysning.fornavn} ${personopplysninger.innsender.opplysning.etternavn} ${visOmInnsenderErGjenlevendeEllerVerge()}`}</BodyShort>
```

Comment:
Vi har en funksjon `formaterNavn` som sikkert er bedre å bruke siden den tar høyde for mellomnavn også. du kan sende inn `personopplysninger.innsender.opplysning` så skal den fungere for alle navnene